Update admin-bro and typeorm

mohammadranjbarz opened the pull request:
related to #572

mohammadranjbarz committed:
Update admin-bro and typeorm, not completed

related to #572

mohammadranjbarz committed:
Fix some of test files syntax errors

mohammadranjbarz committed:
Resolve merge conflicts

mohammadranjbarz committed:
Resolve merge conflicts

mohammadranjbarz committed:
Fix syntax erros in test files

mohammadranjbarz committed:
Update test cases

mohammadranjbarz committed:
Fix project.test.ts test cases

mohammadranjbarz committed:
Fix adminBro.test test cases

mohammadranjbarz committed:
Fix findUserById

mohammadranjbarz committed:
Remove patch-packages, that is not necessary anymore

mohammadranjbarz committed:
Update migrations to be compatible with new version of typeorm

mohammadranjbarz committed:
Resolve merge conflicts

mohammadranjbarz committed:
Fix typeorm connection

mohammadranjbarz committed:
Resolve merge conflicts

mohammadranjbarz committed:
Fix package-lock.json syntax error

mohammadranjbarz committed:
Fix powerroundRepository to be compatible with new typeorm

mohammadranjbarz commented:
@aminlatifi  I updated typeorm and admin-bro, all tests pass but when I opened the admin bro I saw some issues that may take too much time but are necessary before merging

<img width="1440" alt="Screen Shot 1401-06-30 at 16 24 49" src="https://user-images.githubusercontent.com/9850545/191497630-49eb0388-5963-4780-aa46-25e7c93ccf1c.png">

<img width="1440" alt="Screen Shot 1401-06-30 at 16 24 55" src="https://user-images.githubusercontent.com/9850545/191497679-1ee4c957-4187-4c77-9bae-8161f2063989.png">

The relations don't work properly and when opening project page we get error, I think our custom views need to have some changes, so you tell me whether  it worth to put more time to finish that or not

PS: @CarlosQ96  Wants to take a look to see if can find a good solution or not

CarlosQ96 commented:
Ill take a look

aminlatifi committed:
Fixed import adminjs react component issue

aminlatifi committed:
Merged stage into

aminlatifi committed:
Upgraded typeorm and adminjs dependencies - not complete

aminlatifi committed:
Fixed most of issues, only adminjs related remained

aminlatifi committed:
Merge pull request #796 from Giveth/merge_stage_into_adminjs_upgrade

Merge stage into adminjs upgrade

mohammadranjbarz committed:
Fix make projects listed test cases

mohammadranjbarz committed:
Resolve merge conflicts

mohammadranjbarz committed:
Resolve merge conflicts

mohammadranjbarz committed:
Update ts-node-dev to 2.0.0 to fix running problem

CarlosQ96 reviewed (approved):
LGTM, ran tests and verified data wasn't being deleted.

Lets merge develop first then go for staging!

mohammadranjbarz committed:
Resolve merge confclits

mohammadranjbarz committed:
Add some patches to adminjs dependencies

mohammadranjbarz committed:
Add patch file for adminjs dependency

mohammadranjbarz committed:
Update patch of adminjs dependency

mohammadranjbarz committed:
Copy patches folder inside docker image when building it

mohammadranjbarz committed:
Fix editing project from admin bro panel , project edit page

related to #572

mohammadranjbarz committed:
Fix running test cases when remove eager:true for token-organization

related to #572

mohammadranjbarz committed:
Fix syntax error in pre-test-script

mohammadranjbarz committed:
Add joinTable on organization.tokens

aminlatifi committed:
Added build to CI/CD test job

aminlatifi committed:
Merge pull request #808 from Giveth/f_572_update_typeorm_and_adminbro_fix_relations

F 572 update typeorm and adminbro fix relations

aminlatifi committed:
Merge branch 'staging' into f_572_update_typeorm_and_adminbro

# Conflicts:
#	src/resolvers/projectResolver.ts
#	src/utils/validators/projectValidator.ts

mohammadranjbarz committed:
Fix error handling in creating donation in admin bro page

related to #813

related to #809

mohammadranjbarz committed:
Add error handling to related addresses

mohammadranjbarz committed:
Resolve merge conflicts

mohammadranjbarz committed:
Show import third party project error message properly

related to https://github.com/Giveth/impact-graph/issues/572#issuecomment-1400706671

mohammadranjbarz committed:
Show import third party project error message properly

related to https://github.com/Giveth/impact-graph/issues/572#issuecomment-1400706671

mohammadranjbarz committed:
Resolve merge conflicts
